content/support-matrix/delfin/huawei/oceanstor/_index.md

```diff
@@ -0,0 +1,524 @@
+      <tr>
+         <td>Serial number</td>
+         <td>Serial Number</td>
+      </tr>
+      <tr>
+         <td>Manufacturer</td>
+         <td>Manufacturer</td>
+      </tr>
+      <tr>
+         <td>Model</td>
+         <td>Model</td>
+      </tr>
+      <tr>
+         <td>Firmware</td>
+         <td>Firmware Version</td>
+      </tr>
+      <tr>
+         <td>Speed</td>
+         <td>Speed of the disk(rpm)</td>
+      </tr>
+      <tr>
+         <td>Capacity</td>
+         <td>Total number of disk sectors multiply by disk sector size(bytes)</td>
+      </tr>
+      <tr>
+         <td>Status</td>
+         <td>Status</td>
+      </tr>
+      <tr>
+         <td>Physical type</td>
+         <td>Disk physical type. Ex:sata, sas, ssd, etc.</td>
+      </tr>
+      <tr>
+         <td>Logical type</td>
+         <td>Disk logical type. Ex:frontend, maintenance, service, management, etc. </td>
+      </tr>
+      <tr>
+         <td>Health Score</td>
+         <td>Disk health score</td>
+      </tr>
+      <tr>
+         <td>Disk group id</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Location of the disk</td>
+      </tr>
+   </tbody>
+</table>
+
+### Alert data matrix
+
+<table>
+   <thead>
+      <tr>
+         <th>Attribute</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Alert id</td>
+         <td>Identification of alarm</td>
+      </tr>
+      <tr>
+         <td>Alert name</td>
+         <td>Name of the alert</td>
+      </tr>
+      <tr>
+         <td>Severity</td>
+         <td>Severity of the alert</td>
+      </tr>
+      <tr>
+         <td>Category</td>
+         <td>Category of the reported alarm notification. Ex: Fault, Event, Recovery</td>
+      </tr>
+      <tr>
+         <td>Type</td>
+         <td>Type of the alert generated such as CommunicationsAlarm, EquipmentAlarm etc.</td>
+      </tr>
+      <tr>
+         <td>Sequence number</td>
+         <td>Sequence number for the alert, uniquely identifies a
+         given alert instance used for clearing the alert</td>
+      </tr>
+      <tr>
+         <td>Occur time</td>
+         <td>Time at which alert is generated from device. Specified in epoch format</td>
+      </tr>
+      <tr>
+         <td>Description</td>
+         <td>Description</td>
+      </tr>
+      <tr>
+         <td>Recovery advice</td>
+         <td>Some suggestion for handling the given alert</td>
+      </tr>
+      <tr>
+         <td>Resource type</td>
+         <td>Resource type of device/source generating alert.This is always set to "Storage"</td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Location of an alarm. Specified in name/value format</td>
+      </tr>
+   </tbody>
+</table>
+
+### Performance metric data
+
+<table>
+   <thead>
+      <tr>
+         <th>Resource</th>
+         <th>Metric</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
```

Comment:
Table is empty. Remove if not available or applicable.

Reply:
same as above